custom_components/thermal_comfort/config_flow.py

```diff
@@ -21,6 +23,287 @@
 _LOGGER = logging.getLogger(__name__)
 
 
+def get_sensors_by_device_class(
+    _er: EntityRegistry,
+    _hass: HomeAssistant,
+    device_class: SensorDeviceClass,
+    include_all: bool = False,
+) -> list:
+    """Get sensors of required class from entity registry."""
+
+    DEVICE_CLASS_FOR_EXCLUDE = [
+        SensorDeviceClass.AQI,
+        SensorDeviceClass.BATTERY,
+        SensorDeviceClass.CO,
+        SensorDeviceClass.CO2,
+        SensorDeviceClass.CURRENT,
+        SensorDeviceClass.DATE,
+        SensorDeviceClass.ENERGY,
+        SensorDeviceClass.FREQUENCY,
+        SensorDeviceClass.GAS,
+        SensorDeviceClass.ILLUMINANCE,
+        SensorDeviceClass.MONETARY,
+        SensorDeviceClass.NITROGEN_DIOXIDE,
+        SensorDeviceClass.NITROGEN_MONOXIDE,
+        SensorDeviceClass.NITROUS_OXIDE,
+        SensorDeviceClass.OZONE,
+        SensorDeviceClass.PM1,
+        SensorDeviceClass.PM10,
+        SensorDeviceClass.PM25,
+        SensorDeviceClass.POWER_FACTOR,
+        SensorDeviceClass.POWER,
+        SensorDeviceClass.PRESSURE,
+        SensorDeviceClass.SIGNAL_STRENGTH,
+        SensorDeviceClass.SULPHUR_DIOXIDE,
+        SensorDeviceClass.TIMESTAMP,
+        SensorDeviceClass.VOLATILE_ORGANIC_COMPOUNDS,
+        SensorDeviceClass.VOLTAGE,
+    ]
```

Comment:
We could shrink this down if you want
```python
    DEVICE_CLASS_FOR_EXCLUDE = list(SensorDeviceClass)
    DEVICE_CLASS_FOR_EXCLUDE.remove(SensorDeviceClass.TEMPERATURE)
    DEVICE_CLASS_FOR_EXCLUDE.remove(SensorDeviceClass.HUMIDITY)
```

Reply:
Actually it is not what we really want. If some useful for us device class will be added to `SensorDeviceClass`, then we will miss it.